src/nrncvode/nrndaspk.cpp

```diff
@@ -204,24 +204,8 @@ static void do_ode_thread(neuron::model_sorted_token const& sorted_token, NrnThr
     }
 }
 
-static double check(double t, Daspk* ida) {
-    res_gvardt(t, ida->cv_->y_, ida->yp_, ida->delta_, ida->cv_);
-    double norm = N_VWrmsNorm(ida->delta_, ((IDAMem) (ida->mem_))->ida_ewt);
-    Printf("ida check t=%.15g norm=%g\n", t, norm);
-#if 0
-	for (int i=0; i < ida->cv_->neq_; ++i) {
-		printf(" %3d %22.15g %22.15g %22.15g\n", i,
-N_VGetArrayPointer(ida->cv_->y_)[i],
-N_VGetArrayPointer(ida->yp_)[i],
-N_VGetArrayPointer(ida->delta_)[i]);
-	}
-#endif
-    return norm;
-}
-
 int Daspk::init() {
     extern double t;
-    int i;
```

Comment:
The i is used in a #if 0 block below. old line 305. Should be declared there in the for loop.